openshift/topics/introduction.adoc

```diff
@@ -9,12 +9,12 @@
 Red Hat offers multiple OpenShift application templates utilizing the {project_openshift_product_name} image version number {project_version}. These define the resources needed to develop {project_name} {project_version} server based deployment and can be split into the following two categories:
 
 [[passthrough-templates]]
-* Templates using HTTPS and JGroups keystores and a truststore for the {project_name} server, all prepared beforehand. These secure the TLS communication using link:https://docs.openshift.com/container-platform/latest/architecture/networking/routes.html#passthrough-termination[passthrough TLS termination]:
+* Templates using HTTPS and JGroups keystores and a truststore for the {project_name} server, all prepared beforehand. These secure the TLS communication using link:https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html[passthrough TLS termination]:
```

Comment:
There doesn't seem to proper OCP v4.x article describing `passthrough` routes termination AFAICT. So would suggest to point it to `https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html#passthrough-termination`.

That one seems to be the best fit IMHO compared to OCP v4.x pages:
* https://docs.openshift.com/container-platform/4.3/networking/routes/route-configuration.html or
* https://docs.openshift.com/container-platform/4.3/scalability_and_performance/routing-optimization.html

where the `passthrough` mechanism is described only marginally (if even).